src/test/kotlin/com/jayrave/moshi/pristineModels/IntegrationTest.kt

```diff
@@ -0,0 +1,71 @@
+package com.jayrave.moshi.pristineModels
+
+import com.jayrave.moshi.pristineModels.testLib.readFileContent
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.Types
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import java.lang.reflect.Type
+
+class IntegrationTest {
+
+     @Test
+    fun parseNestedJson() {
+
+        val jsonString = "users.json".readFileContent() ?: return Assertions.fail("users.json file not found")
```

Comment:
To preserve uniformity, it would be great if you can use the `jsonString(` test helper which the other tests in `MapperTest.kt` put to use & remove the code flows related to reading from file. 

If reading from file is more desirable, we can let this be too (I don't have a big preference but just wanted to see if they could be kept uniform)

Reply:
Yeah reading from `file.json` was convenient in terms of testing out a large json object instead building objects by hand and then converting it to _jsonString_